src/olympia/api/fields.py

```diff
@@ -414,13 +414,13 @@ def _fetch_some_translations(self, field, langs):
         base_locale = to_language(settings.LANGUAGE_CODE)
         translations = {}
         if base_locale in langs:
-            # we get the base_locale for free - it's just the field text
-            translations[base_locale] = str(field)
-            langs = (lang for lang in langs if lang != base_locale)
+            with override(base_locale):
+                translations[base_locale] = str(field)
```

Comment:
what happens exactly when you `str(<gettext_lazy object>`) in an `override` context?  I'm assuming that `field` will now just be permanently the string value in that locale.  

That's what we want for this function in this case, but it seems like something we'd generally want to avoid because it'd lead to unexpected results.  e.g.
```
field = _('foo')
<'fr' locale is activated by the middleware>
with override('de'):
    value = str(field)
...
print(field)  # would return the 'de' localisation of `foo` rather than the 'fr' localisation.
```

Reply:
Lazy gettext objects should stay lazy, actually.
```python
>>> field = gettext_lazy('All Rights Reserved')

>>> def test():
    ...:     print(field)
    ...:     with override('fr'):
    ...:         print(field)
    ...:     with override('de'):
    ...:         print(field)
    ...:     print(field)
    ...: 

>>> test()
All Rights Reserved
Tous droits réservés
Alle Rechte vorbehalten
All Rights Reserved
```

This makes sense: lazy gettext objects are everywhere in forms, models, admin etc, and the same thread will be re-used for multiple requests, each potentially activating different languages, with the same lazy gettext object every time being re-used and re-translated with whatever the current language is.